test/models/accountSet.ts

```diff
@@ -70,7 +70,7 @@ describe("AccountSet Transaction Verification", function () {
   });
 
   it(`throws w/ invalid EmailHash`, function () {
-    account.EmailHash = 657861645678909876543456789876543;
+    account.EmailHash = 6578656789876543;
```

Comment:
why is this a linter change? and why are semi colons not being removed as per our `prettier` config @natenichols

Reply:
This is a linter change because the number was too big, and the linter said the precision would change on runtime. Since it doesn't matter how big the number is for this check, I just made the numbers smaller.